doc/source/configuration/firewall.rst

```diff
@@ -0,0 +1,37 @@
+.. _firewall:
+
+========
+Firewall
+========
+
+StackHPC Kayobe configuration provides a standardised firewalld configuration.
+The configuration uses the :kayobe-doc:`firewall
+<configuration/reference/hosts.html#firewalld>` host configuration
+functionality of Kayobe.
+
+The firewall configuration is provided in
+``etc/kayobe/inventory/group_vars/all/firewall``. This allows configuration
+variables to be overridden on a per-group or per-host basis (which would not be
+possible for an "extra variable" in ``etc/kayobe/*.yml``). This configuration
+is not used by default, and must be actively opted into. This can be done as
+follows:
+
+.. code-block:: yaml
+   :caption: ``etc/kayobe/controllers.yml``
+
+   controller_firewalld_zones: "{{ stackhpc_firewalld_zones }}"
```

Comment:
Better add the enable flag and default zone here too

Reply:
Done